app/views/welcome/_followers_form.html.haml

```diff
@@ -0,0 +1,17 @@
+%p.text-center
+  %span#follower_new_title_js.title Subscribe to our newsletter
```

Comment:
Classes should be listed before IDs (.title should precede #follower_new_title_js)